Competition2017/src/competition/subsystems/shooter_belt/commands/RunIfAtTargetSpeedCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.shooter_belt.commands;
+
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class RunIfAtTargetSpeedCommand extends BaseCommand {
+
+    final ShooterBeltSubsystem beltSubsystem;
+    public ShooterWheelSubsystem shooterWheelSubsystem;
+    
+    public RunIfAtTargetSpeedCommand(ShooterBeltSubsystem beltSubsystem){
+        this.beltSubsystem = beltSubsystem;
+        this.requires(beltSubsystem);
+    }
+    
+    @Override
+    public void initialize() {
+      
+    }
+
+    @Override
+    public void execute() {
+        if(shooterWheelSubsystem.isAtSpeed() == true){
+            beltSubsystem.updateTelemetry();
```

Comment:
This was the old method where we "did nothing" to let it continue. We should just fix this at the abstract class level and then take that fix in here.